packages/plugin-constraints/sources/Constraints.ts

```diff
@@ -132,6 +122,9 @@ export class Constraints {
     // (Cwd, DependencyIdent, DependencyType, Reason)
     declarations += `gen_invalid_dependency(_, _, _, _) :- false.\n`;
 
+    // (Cwd, Path, Value)
+    declarations += `gen_workspace_field_requirement(_, _, _) :- false.\n`;
```

Comment:
Thinking about it it's too bad the names are so different ... maybe we should uniformize them? ie one of those:

- `gen_enforced_dependency` / `gen_enforced_field`
- `gen_workspace_dependency_requirement` / `gen_workspace_field_requirement`

Reply:
Yeah it irked me as well, I'll change it to `gen_enforced_dependency`/`gen_enforced_field`